package-lock.json

```diff
@@ -55,8 +55,8 @@
         "eslint-config-airbnb-base": "15.0.0",
         "eslint-config-prettier": "8.5.0",
         "eslint-plugin-chai-friendly": "0.7.2",
-        "eslint-plugin-import": "2.26.0",
-        "eslint-plugin-mocha": "10.0.4",
+        "eslint-plugin-import": "2.25.4",
+        "eslint-plugin-mocha": "10.0.3",
```

Comment:
Revert changes in package-lock.json file

Reply:
done